### ISSUE TITLE
Feature request: `asdf env` command that prints out the version info for every tool managed by asdf

### ISSUE THREAD

**Issue by Stratus3D** [asdf-vm/asdf#194]
I'm thinking it might be nice to have a command that can print out all of the information for each tool. We the command would output one line per tool and on each line we would print the tool name, version, place the version was set, and whether or not the version is installed. For example:

```
ruby 2.3.1 (set by ~/project/.ruby-version) installed
erlang 19.3 (set by ~/.tool-versions) not installed
```

This would make it easy to see the environment at a glance. It would also make it easy to see if the tool versions needed for a project you have just cloned are already installed on your system. Right now you can run `asdf install` and see what happens, but you can see what needs to be installed before actually installing the versions.

The `asdf where` and `asdf current` commands already provide similar output, but you must specify the tool name, so they are more limited in that you only see one tool at a time. You can't see all the tools until you run `asdf plugin-list`.

**Comment by Stratus3D** [comment]
@asdf-vm/maintainers what do you think?

**Comment by tuvistavie** [comment]
I agree being able to output all the versions at once could be helpful.
However, instead of increasing the number of commands, what would you think about adding a flag to `asdf current`, for example `asdf current --all` could provide this output.

**Comment by Stratus3D** [comment]
Good idea. I've noticed we have quite a few similar commands, `current`, `where`, and soon, `which`.

I could add the `installed|not installed` field to the existing `current` command then just implement the `--all` flag as a loop over the existing function.

**Comment by tuvistavie** [comment]
Solved by #219
